Add INSECURE_SKIP_HTTPS_VERIFY env var (#146)
signalfx/splunk-extension-wrapper: docs/README.md

|REPORTING_TIMEOUT|`5`|An integer (seconds). Minimum value is 1s.| Specifies metric send operation timeout.                                                                                                                                                                                                                                |
|VERBOSE|`false`|`true` or `false`| Enables verbose logging. Logs are stored in a CloudWatch Logs group associated with a Lambda Function.                                                                                                                                                                  |
|HTTP_TRACING|`false`|`true` or `false`| Enables detailed logs on HTTP calls to Splunk.                                                                                                                                                                                                                          |

> |INSECURE_SSL|`false`|`true` or `false`| Enables skip check validation ssl on HTTP calls to Splunk.                                                                                                                                                                                             |

Reviewer: ```suggestion
|INSECURE_HTTPS|`false`|`true` or `false`| Enables skip certificate validation for HTTPS calls to Splunk.                                                                                                                                                                                             |
```

Is it a call to Splunk Observability Cloud or Splunk Enterprise? AFAIK it is to Splunk Observability Cloud.

> |INSECURE_HTTPS|`false`|`true` or `false`| Enables skip certificate validation for HTTPS calls to Splunk. Commonly used with the SPLUNK_INGEST_URL                                                                                                                                                                                            |

Reviewer 2: Overall this PR looks fine.  My only comment is that this name is ambiguous - I would greatly prefer `INSECURE_SKIP_HTTPS_VERIFY` or similar.  I really don't mean to bikeshed this, but the name simply doesn't indicate what it does.  The chosen name should then flow through the rest of the code (variable names, string constants, etc.).

Author: Done changes @johnbley!
@pellared  I filled the form Splunki CLA.
